developer/debugging/test-bench.md

```diff
@@ -16,57 +16,72 @@ This guide shows how to set up simple test bench that automatically test your co
 
 We will set up a spare machine (bare metal, not a virtual) that will be hosting our experimental Dom0. We will communicate with it via Ethernet and SSH. This tutorial assumes you are familiar with [QubesBuilder](/doc/qubes-builder/) and you have it set up and running flawlessly.
 
-## Setting up the machine
+<div class="alert alert-warning" role="alert">
+  <i class="fa fa-exclamation-triangle"></i>
+  <b>Notice:</b>
+  This setup intentionally weakens some security properties in the testing system. So make sure you understand the risks and use exclusively for testing.
+</div>
```

Comment:
We should avoid using HTML in markdown files directly, it makes it harder to convert to other formats (like PDF) and also to translate. @andrewdavidwong do you remember an alternative for notes like this?

Reply:
> We should avoid using HTML in markdown files directly, it makes it harder to convert to other formats (like PDF) and also to translate. @andrewdavidwong do you remember an alternative for notes like this?

I also remember pointing out this problem and asking for a solution, but I don't recall anyone coming up with a solution.

Reply:
FWIW, I have also been using these alerts throughout the documentation, even though I also hate having HTML in Markdown files. My reasoning is that the value of clarity and preventing users from missing critical information outweighs the cost of having HTML in Markdown files.

Reply:
Yes, I [am aware](https://github.com/QubesOS/qubes-issues/issues/1019#issuecomment-752507119). In double I decided to copy what was already on another page.

Is the following more acceptable? (block quote included)

> **Notice**:
> This setup intentionally weakens some security properties in the testing system. So make sure you understand the risks and use exclusively for testing.

Reply:
I believe I'm done with the proposed changes.